proposals/lip-0041.md

```diff
@@ -165,7 +456,18 @@ This function verifies the nonce of a given transaction `trs` and returns a bool
 
```

Comment:
This comment is rather for the function `verifySignatures` above, which still refers to LIP 17. Shouldn't it refer to the function defined here instead?

Reply:
Good catch. @shuse2 please notice that we changed the name of these two endpoints (maybe not terribly important)